### ISSUE TITLE
Loss did not drop when training VQAD from standard NeRF blender dataset

### ISSUE THREAD

**Issue by cv-dote** [NVIDIAGameWorks/kaolin-wisp#24]
Thanks for this unique library!  
I am using nerf blender data to train the VQAD model.
At first, I used the V8 data and the code worked well.
Then, I tried to use the blender dataset. However, this time, the loss didn't drop. The rendered image is just white and seems to overfit the background.
```
...
2022-08-22 15:20:12,092|    INFO| EPOCH 41/50 | total loss: 1.925E-01 | rgb loss: 1.925E-01
2022-08-22 15:20:22,690|    INFO| EPOCH 42/50 | total loss: 1.914E-01 | rgb loss: 1.914E-01
2022-08-22 15:20:33,307|    INFO| EPOCH 43/50 | total loss: 1.943E-01 | rgb loss: 1.943E-01
2022-08-22 15:20:43,916|    INFO| EPOCH 44/50 | total loss: 1.931E-01 | rgb loss: 1.931E-01
2022-08-22 15:20:54,543|    INFO| EPOCH 45/50 | total loss: 1.927E-01 | rgb loss: 1.927E-01
2022-08-22 15:21:05,190|    INFO| EPOCH 46/50 | total loss: 1.943E-01 | rgb loss: 1.943E-01
2022-08-22 15:21:15,847|    INFO| EPOCH 47/50 | total loss: 1.940E-01 | rgb loss: 1.940E-01
2022-08-22 15:21:26,473|    INFO| EPOCH 48/50 | total loss: 1.938E-01 | rgb loss: 1.938E-01
2022-08-22 15:21:37,133|    INFO| EPOCH 49/50 | total loss: 1.933E-01 | rgb loss: 1.933E-01
2022-08-22 15:21:47,750|    INFO| EPOCH 50/50 | total loss: 1.923E-01 | rgb loss: 1.923E-01
```
Only the dataset part of your vqad_nerf.yaml file is being changed.  
```
dataset:
  dataset_type: "multiview"
  num_rays_sampled_per_img: 4096
  # multiview_dataset_format: "rtmv"
  multiview_dataset_format: "standard"
  mip: 2
  bg_color: "white"
```
My Environment
```
CUDA v11.3
ubuntu 20.04
GeForce RTX 3090
```
Thanks in advance!

**Comment by jiennyteng** [comment]
We also meet similar problem ,when we change another dataset ,and the loss didn't drop.
